tests/bootstrap.php

```diff
@@ -8,6 +8,7 @@
  */
 \Mantle\Testing\manager()
 	->maybe_rsync_plugin()
+	->with_sqlite()
```

Comment:
I was having issues with the testing framework on my PR. I had to re-run the suite to get it to pass. Does using SQLite help alleviate this issue?